_includes/rest/quick-reference.md

```diff
@@ -161,7 +161,7 @@ Whether a request succeeded is indicated by the HTTP status code. A 2xx status c
 
 ## Calling from Client Apps
 
-You should not use the REST API Key in client apps (i.e. code you distribute to your customers). If the Parse SDK is available for your client platform, we recommend using our SDK instead of the REST API. If you must call the REST API directly from the client, you should use the corresponding client-side Parse key for that plaform (e.g. Client Key for iOS/Android, or .NET Key for Windows/Xamarin/Unity).
+You should not use the REST API Key in client apps (i.e. code you distribute to your customers). If the Parse SDK is available for your client platform, we recommend using our SDK instead of the REST API. If you must call the REST API directly from the client, you should use the corresponding client-side Parse key for that plaform (e.g. Client Key for iOS/Android, or .NET Key for Windows/XAMARIN/.NET MAUI/Unity).
```

Comment:
We've removed support for Xamarin with the .NET SDK 4 release, so we can also remove any Xamarin reference from the docs.

```suggestion
You should not use the REST API Key in client apps (i.e. code you distribute to your customers). If the Parse SDK is available for your client platform, we recommend using our SDK instead of the REST API. If you must call the REST API directly from the client, you should use the corresponding client-side Parse key for that plaform (e.g. Client Key for iOS/Android, or .NET Key for Windows/.NET MAUI/Unity).
```